bin/configurer/utils/generate-local-user-mappings.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+SPRING_PROFILES=user-mappings
```

Comment:
That could be inlined as for me there is no sufficient reason to have it here. It won't be changed in the future and significance of this in context of file generation is minor IMO.

Reply:
yes makes sense :)